tic-tac-toe/ticTacToe.js

```diff
@@ -0,0 +1,207 @@
+// wait for the DOM to finish loading
+$(document).ready(function () {
+  
+  var count = 0;
+  var board = [1,1,1,1,1,1,1,1,1]; //sets up initial array
+
+  console.log('javascript is working');
+  
+
+
+
+  $('.box').click(function () {
+  	//handles clicks on each box
+  	//calls other game functionality
+
+  	count = count+1;
+  	console.log(count);
+  	// print id
+  	var elemID = $(this).attr('id');
+  	console.log('#'+elemID);
+  	$('#'+elemID).text(turn);
+  		callPlayer();
+  		fillBoard();
+  		//checkForWinner();
+
+  });
+  
+  // all code to manipulate the DOM
+  // goes inside this function
+  
+  
+  var turn = function () {  //decides who's turn it is, X or O
+    if (count % 2 === 0) {
+    	return 'O';
+    }
+    else {
+    	return 'X';
+    }
+  };
+
+  var callPlayer = function() {
+  	if (count % 2 === 0) {
+  		$('h3').text("Player One, Go!");
+  		console.log("player 1 goes");
+  	} else {
+  		$('h3').text('Player Two, Go!');
+  		console.log("player 2 goes");
+  	}
+};
+
+//MAKE RESET BUTTON WORK
+$('button').click(function () {
+	location.reload();
+});
+
+
+//pushes x or o into array based on ID of square 
+  var fillBoard = function () {
+  	var board = board[(elemID-1)].push(turn);
```

Comment:
elemID is defined in a different function so it's out of scope here.  You'll have to either pass it to this function as an argument OR re-find it in this function.